frontend/src/pages/instance/Settings/Palette.vue

```diff
@@ -72,7 +72,14 @@ export default {
         }
     },
     computed: {
-        ...mapState('account', ['team', 'teamMembership']),
+        ...mapState('account', ['team', 'teamMembership', 'features']),
+        catalogFeatureEnabledForTeam () {
+            if (!this.features.customCatalogs) {
+                return false
+            }
+            const flag = this.team.type.properties.features?.customCatalogs
+            return flag === undefined || flag
+        },
```

Comment:
This is the instance-specific view of settings, so we have to check both the platform feature flag and the current team type features.